ios/RNCAsyncStorage.m

```diff
@@ -16,9 +16,25 @@
 #import "RCTLog.h"
 #import "RCTUtils.h"
 
+typedef NS_ENUM(NSInteger, RCTStorageLocation) {
+  Documents,
```

Comment:
Can we conform Apple naming style? 
<img width="680" alt="image" src="https://user-images.githubusercontent.com/5061845/54329551-175e6880-464d-11e9-8be3-233767db3228.png">